src/components/Badge/Badge.scss

```diff
@@ -44,28 +44,32 @@
     color: $LabelColor;
   }
   &.nitrozen-badge-default-fill {
-    background: $Zambezi;
-    border: 0.1rem solid $Zambezi;
+    background: $JDSDefaultLightColor;
+    color: $JDSDefaultColor;
+    // border: 0.1rem solid $Zambezi;
   }
   &.nitrozen-badge-info-fill {
     background: $PrimaryColor;
     border: 0.1rem solid $PrimaryColor;
   }
   &.nitrozen-badge-success-fill {
-    background: $SuccessColor;
-    border: 0.1rem solid $SuccessColor;
+    background: $JDSSuccessLightColor;
```

Comment:
The same properties are repeated, please consider the use of _mixins_ and _include_